pages/maintenance.md

```diff
@@ -15,18 +15,26 @@ noMegamenu: true
     <div class="row">
       <div class="small-12 columns">
         <div style="padding: 2em 0;">
-        <h3>We're sorry. The page you're looking for is currently down while we fix a few things. We'll be back online as soon as we can.</h3>
+        <h1>Site maintenance</h1>
+        <div>
+          <p>
+            We're working on VA.gov right now. This maintenance will last about 3 hours.
+          </p>
+          <p>
+            During this time, you won’t be able to sign in, use tools, or visit VA.gov webpages. Thank you for your patience.
+          </p>
+          <p><strong>Date:</strong> Wednesday, January 31, 2024</p>
+          <p><strong>Start and end time:</strong> 2:00 a.m. to 5 a.m. ET</p>
+          <h2>Need help during this time?</h2>
+          <ul>
+            <li><strong>If you need help managing your VA benefits or connecting with your local VA medical center,</strong> call us at <a href="tel:+18006982411" aria-label="8 0 0. 6 9 8. 2 4 1 1.">800-698-2411</a><a href="tel:711" aria-label="TTY. 7 1 1."> (TTY: 711)</a>. We're here 24/7.</li>
+            <li><strong>If you’re in crisis or having thoughts of suicide,</strong> call our Veterans Crisis Line at <a href="tel:988" aria-label="988">988</a> and select 1. We offer confidential support anytime, day or night.</li>
+            <li><strong>If you think your life or health is in danger, call <a href="tel:911" aria-label="911">911</a>,</strong> or go to the nearest emergency room.</li>
+          </ul>
         </div>
       </div>
     </div>
   </div>
 </div>
 
 <!-- Maintenance Page End -->
-
```

Comment:
Hmm, do these empty lines need to be here? My editor automatically deletes trailing white space and extra new lines.